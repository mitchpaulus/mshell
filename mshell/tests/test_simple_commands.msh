[ls --color=auto] ;
[echo hello-world] ;
