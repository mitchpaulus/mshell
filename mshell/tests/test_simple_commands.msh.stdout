if_test.msh
if_test.msh.stdout
test_file.sh
test.sh
test_simple_commands.msh
test_simple_commands.msh.stdout
hello-world
